lib/packwerk.rb

```diff
@@ -39,10 +40,6 @@ module OutputStyles
     autoload :Plain
   end
 
-  autoload_under "commands" do
-    autoload :OffenseProgressMarker
-  end
-
```

Comment:
This does not seem necessary, and was actually interfering with the `autoload :Commands` that we added above.

Reply:
👍 

Seems it was left behind in https://github.com/Shopify/packwerk/pull/124